Cython/Utility/Coroutine.c

```diff
@@ -718,14 +722,21 @@ PyObject *__Pyx_Coroutine_SendEx(__pyx_CoroutineObject *self, PyObject *value, i
     // - do not touch external frames and tracebacks
 
     exc_state = &self->gi_exc_state;
-    if (exc_state->exc_type) {
+    if (exc_state->exc_value) {
         #if CYTHON_COMPILING_IN_PYPY
         // FIXME: what to do in PyPy?
         #else
         // Generators always return to their most recent caller, not
         // necessarily their creator.
-        if (exc_state->exc_traceback) {
-            PyTracebackObject *tb = (PyTracebackObject *) exc_state->exc_traceback;
+        PyObject *exc_tb;
+        #if PY_VERSION_HEX >= 0x030B00a4
+        // owned reference!
+        exc_tb = PyException_GetTraceback(exc_state->exc_value);
+        #else
+        exc_tb = exc_state->exc_traceback
```

Comment:
Does this need an incref?

Reply:
I don't think it does – unless `PyThreadState_GetFrame()` can trigger a context switch, but I doubt it.